redash/cli/rq.py

```diff
@@ -22,3 +25,17 @@ def worker(queues='default'):
     with Connection(redis_connection):
         w = Worker(queues)
         w.work()
+
+
+@manager.command()
+def healthcheck():
```

Comment:
@arikfr thoughts on this? This command will exit with 0 if all workers running on this machine have had a heartbeat in the last 60 seconds.

Reply:
I wonder if this logic makes sense or whether we should have an option to specify a single worker :thinking:

Reply:
Assuming you would typically run a single worker per container - it makes sense, because a non-zero exit code would probably indicate that you need to restart the container.

Given multiple workers in a container, you would probably want to return the PIDs of the misbehaving ones so you could kill and restart them, but I'm not sure how well that plugs into things like ECS's (or other services) healthchecks.